client/scripts/views/components/sidebar/governance_section.tsx

```diff
@@ -308,6 +333,10 @@ export class GovernanceSection
         : false,
       onclick: (e, toggle: boolean) => {
         e.preventDefault();
+        if (isCommandClick(e)) {
```

Comment:
Ah! so this could be DRY'd out even more. 
The pattern seems to be: 
``` 
if statement (command click statement)
     nested window.open with [destination]
else
navigate to [destination]
optional "callback" to toggle sidebar.
```
I see this as a helper method that takes in a destination and optional sidebar callback that can be run in a single line, rather than having this entire pattern copy/pasta across the sidebar.

Reply:
Ah true, will change it